app/controllers/api/v1/organization_pages_controller.rb

```diff
@@ -1,33 +1,17 @@
 class Api::V1::OrganizationPagesController < Api::ApiController
-  require_authentication :update, :create, :destroy, scopes: [:organization]
-
-  resource_actions :default
-
-  schema_type :strong_params
-
-  allowed_params :create, :url_key, :title, :content, :language
-  allowed_params :update, :url_key, :title, :content, :language
+  include Pages
 
-  before_filter :set_language_from_header, only: [:index]
-
-  def controlled_resources
-    @controlled_resouces ||= super.where(organization: params[:organization_id])
-  end
-
-  protected
+  require_authentication :update, :create, :destroy, scopes: [:organization]
```

Comment:
Everything else is extracted (to the Pages concern). That particular line seems to run in a different context (self=concern vs. self=controller). I've got it working, but I'm going to have another look first thing tomorrow and see if I can't de-awkward it a bit.